server/selinux/server/pulp-celery.te

```diff
@@ -40,8 +47,17 @@ allow celery_t self:unix_dgram_socket { create connect };
 #
 
 allow celery_t self:process { setsched };
-allow celery_t tmp_t:dir { write remove_name create add_name rmdir setattr };
-allow celery_t tmp_t:file { rename create unlink setattr write read getattr open lock };
+
+allow celery_t celery_tmp_t:dir { write remove_name create add_name rmdir setattr };
+files_tmp_filetrans(celery_t, celery_tmp_t, file)
+
+allow celery_t celery_tmp_t:file { rename create unlink setattr write read getattr open lock };
```

Comment:
I think this statement should be:

`allow celery_t celery_tmp_t:file manage_file_perms;`

I looked up what that macro expands into in `/usr/share/selinux/devel/include/support/obj_perm_sets.spt` and it says: { create open getattr setattr read write append rename link unlink ioctl lock }

This expands the permissions some (append, ioctl, link), but for tmp files (celery_tmp_t) context I this is the right thing to do. This needs to be checked that it compiles on FC20, EL6, and EL7.